Bindings/VisibilityConverter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Bindings
+{
+    class VisibilityConverter : IValueConverter
```

Comment:
The most common convention is to name it `BoolToVisibilityConverter`. Including the types in the name is not pretty but there is not much the compiler can check given the signatures of the methods.